script.js

```diff
@@ -22,4 +22,141 @@ closeAbout.addEventListener('click', () => {
 const closeContact = document.querySelector('.Contact');
 closeContact.addEventListener('click', () => {
   modalMenu.style.left = '-100%';
-});
\ No newline at end of file
+});
+
+const projects = [
+  {
+    title: 'Tonic',
+    details: ['CANOPY', 'Back End Dev', '2015'],
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tec: ['html', 'css', 'javaScript'],
+    image: './Images/Works/SnapshootPortfolio1.png',
+    link: 'https://www.github.com',
+    linkLive: 'https://alejandrakeber.github.io/',
+    linkUrl: 'https://github.com/AlejandraKeber/Project-setup-and-mobile-main',
+  },
+  {
+    title: 'Mutli-Post Stories',
+    details: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    tec: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    image: './Images/Works/SnapshootPortfolio2.png',
+    link: 'https://www.github.com',
+    linkLive: 'https://alejandrakeber.github.io/',
+    linkUrl: 'https://github.com/AlejandraKeber/Project-setup-and-mobile-main',
+  },
+  {
+    title: 'Facebook 360',
+    details: ['CANOPY', 'Full Stack Dev', '2015'],
+    description: 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    tec: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    image: './Images/Works/SnapshootPortfolio3.png',
+    link: 'https://www.github.com',
+    linkLive: 'https://alejandrakeber.github.io/',
+    linkUrl: 'https://github.com/AlejandraKeber/Project-setup-and-mobile-main',
+  },
+  {
+    title: 'Uber Navigation',
+    details: ['Uber', 'Lead Developer', '2018'],
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    tec: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    image: './Images/Works/SnapshootPortfolio4.png',
+    link: 'https://www.github.com',
+    linkLive: 'https://alejandrakeber.github.io/',
+    linkUrl: 'https://github.com/AlejandraKeber/Project-setup-and-mobile-main',
+  },
+];
```

Comment:
- [ ] Please as mentioned by the previous reviewer, your popup design needs to stick as much as possible to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), the mobile version and desktop version 🙃 🙃

> |     | Your desktop popup design                           | Figma desktop des.                  |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | <img width="836" alt="Screen Shot 2022-09-08 at 21 52 18" src="https://user-images.githubusercontent.com/74501523/189216782-6b324bad-a1e2-4f11-9152-0ae1de0cfb62.png"> | <img width="577" alt="Screen Shot 2022-09-08 at 21 50 53" src="https://user-images.githubusercontent.com/74501523/189216842-dea8f0de-4aa8-4339-be38-ec547b3cb360.png"> |
> |     |  Your mobile popup design                        |  Figma mobile popup design
> |     | <img width="322" alt="Screen Shot 2022-09-08 at 21 56 37" src="https://user-images.githubusercontent.com/74501523/189217105-28ddca34-4081-4090-8aeb-36db8bea0f89.png"> | <img width="322" alt="Screen Shot 2022-09-08 at 21 57 52" src="https://user-images.githubusercontent.com/74501523/189217206-f314db86-f4dc-46ff-9e81-915bf3d684b0.png">     |

- [ ] Please kindly fix your design 🙃 😉

Reply:
Hi @cynthiainga I sent you a message in Slack.
It has been a very hard project for me
Other partners told me they used the same images for desktop and mobile versions and their projects were approved
I don't know how to use different images ... can you give a hint, please?
Any suggestion to fix the width of the popup and the blur of the main page?
I have already received 2 reviews and I have only one chance left
Thank you!